java/dataproc-wordcount/pom.xml

```diff
@@ -32,15 +32,11 @@
   </parent> -->
 
   <properties>
+    <bigtable.version>0.9.5</bigtable.version>
     <bigtable.projectID>YOUR_PROJECT_ID_HERE</bigtable.projectID>
     <bigtable.instanceID>YOUR_INSTANCE_ID_HERE</bigtable.instanceID>
 
-    <bigtable.version>0.9.4</bigtable.version>
-    <hbase.version>1.2.1</hbase.version><!-- may need to match Dataproc HBase version -->
-    <hbase.if>1.2</hbase.if> <!-- hbase interface - it changed between 1.0 and 1.1.x -->
-
-    <hadoop.version>2.4.1</hadoop.version>
-    <compat.module>hbase-hadoop2-compat</compat.module>
+    <bigtable.version>0.9.5-SNAPSHOT</bigtable.version>
```

Comment:
There's a duplicate <bigtable.version> tag.  I think we should wait until 0.9.5 to commit this change.